config/config.json

```diff
@@ -125,8 +125,9 @@
         "EmailBatchingInterval": 30
     },
     "RateLimitSettings": {
-        "EnableRateLimiter": true,
+        "Enable": false,
```

Comment:
Looks like you missed a couple changes to the config file, can you revert these as well?